admin_manual/configuration_files/previews_configuration.rst

```diff
@@ -101,8 +101,40 @@ to 'null':
 JPEG quality setting:
 ^^^^^^^^^^^^^^^^^^^^^
 
-Default JPEG quality setting for preview images is '90'. Change this with:
+Default JPEG quality setting for preview images is '80'. Change this with:
 
 :: 
 
   occ config:app:set preview jpeg_quality --value="60"
+
+Imaginary:
+^^^^^^^^^^^^^^^^^^^^^
+
+If you want to enable imaginary, set the url.
+Also requires the ``OC\Preview\Imaginary`` provider to be enabled:
+
+::
+
+  <?php
+    'preview_imaginary_url' => 'url',
+
+If you want set the preview format for imaginary.  
+You can change between jpeg and webp, the default is jpeg:
+
+::
+
+  <?php
+    'preview_format' => 'webp',
+
+If you want set a api key for imaginary':
+
+::
+
+  <?php
+    'preview_imaginary_key' => 'secret',
```

Comment:
maybe we could merge this somehow with https://github.com/nextcloud/documentation/blob/607b87830f0a01698119f900ee3edffeeb4d0cb2/admin_manual/installation/server_tuning.rst#previews so that we dont duplicate the documentation?

Reply:
Yes, preview_imaginary_url would be double.
I put the 3 options in the file server_tuning.rst.